src/mumble/LookConfig.cpp

```diff
@@ -431,3 +433,14 @@ void LookConfig::on_qcbUsersAlwaysVisible_stateChanged(int state) {
 	// Only enable the user visibility timeout settings when all users are not always visible
 	qsbSilentUserLifetime->setEnabled(!usersAlwaysVisible);
 }
+
+void LookConfig::on_qcbTalkingUITransparent_stateChanged(int state) {
+	// Handle transparency checkbox state change
+	qsbTransparencyLevel->setEnabled(state == Qt::Checked);
+}
```

Comment:
In new code, we want to move away from implicit event connections by method name like this.

Please refactor this to use a plausible name like ``toggleTransparency`` or something and use the ``QObject::connect()`` method to explicitly connect the event.
Should look something like
``QObject::connect(qcbTalkingUITransparent, QComboBox::stateChanged, this, LookConfig::toggleTransparency)`` (untested)